.github/workflows/build.yml

```diff
@@ -32,13 +32,18 @@ jobs:
           role: endlessm-eos-build-meta
           pki: |
             pki/issue/endless-client {"common_name": "github@endlessos.org", "ttl": "12h"} ;
+          secrets: |
+            secret/ostree-builder/gpg eoask1 | OSTREE_BUILDER_GPG_KEY
+            secret/ostree-builder ssh | OSTREE_BUILDER_SSH_KEY
```

Comment:
The `eoask1` and `gpg` are part of the secret path, but the part needed is the `private` field in both values. Like:

```
secrets: |
  secret/ostree-builder/gpg/eoask1 private | OSTREE_BUILDER_GPG_KEY ;
  secret/ostree-builder/ssh private | OSTREE_BUILDER_SSH_KEY ;
```

https://github.com/hashicorp/vault-action/blob/main/README.md#example-usage also suggests you want to end each line with `;`. I presume that's needed for correct parsing. Also... I need to update the policy to allow reading `secret/ostree-builder/gpg/eoask1`. Hang on there.

Reply:
This is now working.